src/protocol.c

```diff
@@ -104,10 +104,19 @@ int protocol_supports_flag(struct protocol *proto, uint flag)
 			return 1;
 
 		/* run a live check */
-		ret = _sock_supports_reuseport(proto->fam, proto->sock_type, proto->sock_prot);
-		if (!ret)
+		if (global.tune.options & GTUNE_NO_MPTCP){
+			ret = _sock_supports_reuseport(proto->fam, proto->sock_type, proto->sock_prot_fb);
```

Comment:
if you are not on `__linux__`, `proto->sock_prot_fb` will not be defined.

I guess you should at least have: `if (proto->sock_prot_fb > 0 && global.tune.options & GTUNE_NO_MPTCP)` (or a `#ifdef __linux__` but easier to avoid these `ifdef` I think)